x-pack/solutions/observability/plugins/streams_app/public/components/stream_detail_lifecycle/metadata.tsx

```diff
@@ -229,7 +231,7 @@ function MetadataRow({
     <EuiFlexGroup alignItems="center" gutterSize="xl" responsive={false}>
       <EuiFlexItem grow={1}>
         <EuiText>
-          <b>{metadata}</b>
+          <h5>{metadata}</h5>
```

Comment:
In combination with heading tags, it should use the EuiTitle component and not EuiText